kubetest/kops.go

```diff
@@ -53,6 +53,8 @@ var (
 	kopsBaseURL      = flag.String("kops-base-url", "", "Base URL for a prebuilt version of kops")
 	kopsVersion      = flag.String("kops-version", "", "URL to a file containing a valid kops-base-url")
 	kopsDiskSize     = flag.Int("kops-disk-size", 48, "Disk size to use for nodes and masters")
+
+	kopsPublishVersion = flag.String("kops-publish-version", "", "Publish kops version to the specified gs:// path on success")
```

Comment:
Would `--kops-publish` or `--kops-publish-path` make more sense here?

Reply:
Very open to either.

I had `--kops-publish` originally, which mirrored `--publish`.  I thought it was a little ambiguous though, in that it sounded like we were publishing kops itself (copying it to a known location, I guess), rather than just writing a version file.